homeassistant/components/sensor/swiss_hydrological_data.py

```diff
@@ -19,12 +19,14 @@
 REQUIREMENTS = ['beautifulsoup4==4.4.1']
```

Comment:
Why not take it 1 step further and drop the beautifulsoup4 requirement and use `xmltodict` like the YR sensor does?